packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -42,8 +43,14 @@ extension InspectorColorScheme on ColorScheme {
 const double iconPadding = 5.0;
 const double chartLineStrokeWidth = 1.0;
 const double columnWidth = 16.0;
-const double verticalPadding = 10.0;
-const double rowHeight = 24.0;
+
+double verticalPadding() {
+  return 10.0 * fontSizeFactorFromTheme();
+}
+
+double rowHeight() {
```

Comment:
these should be getters not methods. 
Only use a method for something that is conceptually a getter if the operation performed is very slow.